contracts/seed/Seed.sol

```diff
@@ -44,7 +44,8 @@ contract Seed {
 
     bytes   public metadata;           // IPFS Hash
 
-    uint256 constant internal PRECISION = 10 ** 18;  // 0% = 0; 1% = 10 ** 16; 100% = 10 ** 18
+    uint256 constant internal PRECISION = 10 ** 18;  // 1 token = 10**18, 0.01 token = 10**16, 0 token = 0
+    uint256 constant internal PERCENT_PRECISION = 10 ** 20 // 100*PRECISION
```

Comment:
Hmm, now I have a question, what do you mean be PERCENT_PRECISION?

Reply:
hm, because it helps in calculating percent as well as helps with precision. Should I rename it to FEE_PRECISION?